core/src/main/scala/better/files/File.scala

```diff
@@ -968,14 +968,14 @@ class File private (val path: Path)(implicit val fileSystem: FileSystem = path.g
       Files.walkFileTree(
         path,
         new SimpleFileVisitor[Path] {
-          def newPath(subPath: Path): Path = destination.path.resolve(path.relativize(subPath))
+          def newPath(subPath: Path | Null): Path = destination.path.resolve(path.relativize(subPath))
```

Comment:
**Nullable argument**
Argument that can be null and should therefore have a `| Null` type.